### PR TITLE
Data dicos

### PR DESCRIPTION
**Pull request by ylimerm3**
things i did in this PR:
-add "_b" for STIAS/SPS6 version b scrd/perc line in the data dico for that questionnaire
-upload the data dicos for central tracker and psychopy task
-delete my previously uploaded data dicos that were in the wrong format (xlsx instead of csv)
-update the readme as the project goals have changed since i first created it

### CODE REVIEW

File: data-monitoring/data-dictionary/of-b_datadict.csv [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿variable,description,detail,allowedSuffix,valueType,allowedValues,valueInfo,naValueType,activeStatus,replacement,provenance
+participant,participant id,participant id that is manually entered at start of task,s1_r1_e1,integer,"[240000,249999]",NA,NA,1,NA,direct-psychopy
+which button yes,counterbalance,participant counterbalance that is manually entered at start of task,s1_r1_e1,char,"""left"",""right""","""left"", different=left | ""right"", different=right",NA,1,NA,direct-psychopy
```
Comment (jessb0t) [inline data-monitoring/data-dictionary/of-b_datadict.csv:3]: 1. Is the variable name really "which button yes" (with spaces)?
2. I'm not clear on this part: `"left", different=left | "right", different=right`?

File: data-monitoring/data-dictionary/of-b_datadict.csv [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+participant,participant id,participant id that is manually entered at start of task,s1_r1_e1,integer,"[240000,249999]",NA,NA,1,NA,direct-psychopy
+which button yes,counterbalance,participant counterbalance that is manually entered at start of task,s1_r1_e1,char,"""left"",""right""","""left"", different=left | ""right"", different=right",NA,1,NA,direct-psychopy
+date,date,date when task was run. auto-populated when task run,s1_r1_e1,date,NA,NA,NA,1,NA,direct-psychopy
+psychopyVersion,psychopy version,psychopy version used to run experiment. auto-populated when task run,s1_r1_e1,char,version number,NA,NA,1,NA,direct-psychopy
+frameRate,frame rate,framerate used to run experiment. auto-populated when task run,s1_r1_e1,real,"[0,inf]",NA,NA,1,NA,direct-psychopy
+trials_4.thisN,trial number (practice),denotes trial number in the practice block,s1_r1_e1,integer,"[0,83]",NA,NA,1,NA,direct-psychopy
+key_resp_12.keys,response values (practice),vector of response values corresponding to all responses made during the response window on the current trial of practice,s1_r1_e1,vector of char,"""1"",""8""",NA,NA,1,NA,direct-psychopy
```
Comment (jessb0t) [inline data-monitoring/data-dictionary/of-b_datadict.csv:8]: So only 1s and 8s?

File: data-monitoring/data-dictionary/of-b_datadict.csv [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+frameRate,frame rate,framerate used to run experiment. auto-populated when task run,s1_r1_e1,real,"[0,inf]",NA,NA,1,NA,direct-psychopy
+trials_4.thisN,trial number (practice),denotes trial number in the practice block,s1_r1_e1,integer,"[0,83]",NA,NA,1,NA,direct-psychopy
+key_resp_12.keys,response values (practice),vector of response values corresponding to all responses made during the response window on the current trial of practice,s1_r1_e1,vector of char,"""1"",""8""",NA,NA,1,NA,direct-psychopy
+key_resp_12.corr,trial accuracy (practice),accuracy of the first response provided on a practice trial,s1_r1_e1,logical,"0,1",0=error | 1=correct,NA,1,NA,direct-psychopy
+key_resp_12.rt,reaction time values (practice),vector of reaction times corresponding to all responses made during the response window on the current trial of practice,s1_r1_e1,vector of real,"[0,inf]",NA,NA,1,NA,direct-psychopy
+accuracy,practice overall accuracy,averaged accuracy during the practice denoted by a single value,s1_r1_e1,real,"[0,100]",NA,NA,1,NA,direct-psychopy
```
Comment (jessb0t) [inline data-monitoring/data-dictionary/of-b_datadict.csv:11]: Confirming this is really between 0 and 100 (not 0 and 1)?

File: data-monitoring/data-dictionary/of-b_datadict.csv [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+trials_5.thisTrialN,block number (main),denotes block number in the main task,s1_r1_e1,integer,"[0,15]",NA,NA,1,NA,direct-psychopy
+trials.thisN,trial number (main),denotes trial number within a single block in the main task,s1_r1_e1,integer,"[0,83]",NA,NA,1,NA,direct-psychopy
+key_resp_6.keys,response values (main),vector of response values corresponding to all responses made during the response window on the current trial of main task,s1_r1_e1,vector of char,,NA,NA,1,NA,direct-psychopy
```
Comment (jessb0t) [inline data-monitoring/data-dictionary/of-b_datadict.csv:14]: Looks like this row is missing the allowedValues data.

File: data-monitoring/data-dictionary/of-b_datadict.csv [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+key_resp_6.corr,trial accuracy (main),accuracy of the first response provided on a main trial,s1_r1_e1,logical,"0,1",0=error | 1=correct,NA,1,NA,direct-psychopy
+key_resp_6.rt,reaction time values (main),vector of reaction times corresponding to all responses made during the response window on the current trial of main task,s1_r1_e1,vector of real,"[0,inf]",NA,NA,1,NA,direct-psychopy
+isi,isi,"inter stimulus interval, corresponding to the time in milliseconds between the two stimuli",s1_r1_e1,real,"[400,700]",NA,NA,1,NA,direct-psychopy
+iti,iti,"inter trial interval, also known as response stimulus interval, corresponding to the time in milliseconds between the response of the current trial and first stimulus of the next trial",s1_r1_e1,real,"[1,1.3]",NA,NA,1,NA,direct-psychopy
```
Comment (jessb0t) [inline data-monitoring/data-dictionary/of-b_datadict.csv:18]: Wow!  Between 1 and 1.3 milliseconds?!  I have no personal experience by which to benchmark, but that seems really fast, so I'm just triple-checking that it isn't a typo.

File: data-monitoring/data-dictionary/of-b_datadict.csv
Comment (jessb0t) [inline data-monitoring/data-dictionary/of-b_datadict.csv:1]: This is so awesome, @ylimerm3.  If I were wanting to use your task or data, I would find your descriptions here really invaluable!

File: README.md
Comment (jessb0t) [inline README.md:1]: Overall, this looks great!  One little thing, that you could not have known from the samples I provided: in testing Valence, we found that "_timestamp" was not as reliable a datapoint as "_complete," so we switched that.  This affects the "detail" column of all the REDCap survey questions.  So: `"demo_c_s1_r1_e1_timestamp"!=NULL` should become `"demo_c_s1_r1_e1_complete"!=NULL`.

**Review by jessb0t — APPROVED** [review]
@ylimerm3 I added some notes for you to address, but went ahead and approved so you can merge whenever you are able to get to those.  Let me know if you have any questions about anything!

File: data-monitoring/data-dictionary/of-b_datadict.csv (hunk 5 above)
Comment (ylimerm3) [reply data-monitoring/data-dictionary/of-b_datadict.csv:18]: i meant 1-1.3 seconds. thank you for catching this

File: data-monitoring/data-dictionary/of-b_datadict.csv (hunk 1 above)
Comment (ylimerm3) [reply data-monitoring/data-dictionary/of-b_datadict.csv:3]: 1. yes
2. ok i can make it more descriptive

File: data-monitoring/data-dictionary/of-b_datadict.csv (hunk 2 above)
Comment (ylimerm3) [reply data-monitoring/data-dictionary/of-b_datadict.csv:8]: yes

File: data-monitoring/data-dictionary/of-b_datadict.csv (hunk 3 above)
Comment (ylimerm3) [reply data-monitoring/data-dictionary/of-b_datadict.csv:11]: yes it's coded differently

File: data-monitoring/data-dictionary/of-b_datadict.csv (hunk 4 above)
Comment (ylimerm3) [reply data-monitoring/data-dictionary/of-b_datadict.csv:14]: thank you for catching this

File: README.md
Comment (ylimerm3) [reply README.md:1]: got it, thanks

**Pull request merged**